editor.planx.uk/src/@planx/components/MapAndLabel/Public/CopyFeature.tsx

```diff
@@ -27,39 +29,56 @@ export const CopyFeature: React.FC<Props> = ({
     (_, sourceIndex) => sourceIndex !== destinationIndex,
   );
 
+  const formik = useFormik({
+    initialValues: {
+      sourceLabel: sourceFeatures[0]?.properties?.label,
+    },
+    onSubmit: ({ sourceLabel }) => copyFeature(sourceLabel, destinationIndex),
+  });
+
   return (
-    <Box>
-      <InputLabel label="Copy from" id={`select-${destinationIndex}`}>
-        <SelectInput
-          disabled={isDisabled}
-          aria-describedby="copy-feature-description"
-          bordered
-          required
-          title={"Copy from"}
-          value={""}
```

Comment:
This line was the cause of the keyboard access problem. As there's no value, the MUI select doesn't automatically grant focus to the child correctly. Once I added a value here, keyboard access worked as expected.

I expanded this to use Formik as it soon became apparent that we'd need to handle change/blur/updating value etc and this actually saves a fair amount of boilerplate.